### PR TITLE
Madgraph Cards for Signal, Background and SBI samples for Off-Shell HToWW process 

### PR DESCRIPTION
**Pull request by sv3048**
Dear experts!

I have included proc and run cards for ggF HToWWToLNuQQ off-shell processes (S,B,SBI). In order to decay both Ws in final state to LNuQQ, Pythia is being used. I want to do private sample generation using the gridpacks produced through these cards. Please help me in tweaking these run cards if needed.

Thanks and Regards,
Sadhana Verma

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/GluGluHToWW_OffShell/ggWW_off_shell/ggWW_off_shell_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+     1        = lpp1    ! beam 1 type 
+     1        = lpp2    ! beam 2 type
+     6500.0     = ebeam1  ! beam 1 total energy in GeV
+     6500.0     = ebeam2  ! beam 2 total energy in GeV
+# To see polarised beam options: type "update beam_pol"
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+     nn23lo1    = pdlabel     ! PDF set                                     
+     230000    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/GluGluHToWW_OffShell/ggWW_off_shell/ggWW_off_shell_run_card.dat:47]: please use the following lines for the pdf:
'lhapdf'    = pdlabel     ! PDF set                                  
 $DEFAULT_PDF_SETS    = lhaid    
 $DEFAULT_PDF_MEMBERS    = reweight_PDF  ! if pdlabel=lhapdf, this is the lhapdf number

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/GluGluHToWW_OffShell/ggWW_off_shell/ggWW_off_shell_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+#*********************************************************************
+# Store info for systematics studies                                 *
+# WARNING: Do not use for interference type of computation           *
+#*********************************************************************
+#
+   True  = use_syst      ! Enable systematics studies
+#
+#isystematics = systematics_program ! none, systematics [python], SysCalc [depreceted, C++]
+#['--mur=0.5,1,2', '--muf=0.5,1,2', '--pdf=errorset'] = systematics_arguments ! see: https://cp3.irmp.ucl.ac.be/projects/madgraph/wiki/Systematics#Systematicspythonmodule
+# Syscalc is deprecated but to see the associate options type'update syscalc'
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/GluGluHToWW_OffShell/ggWW_off_shell/ggWW_off_shell_run_card.dat:282]: you can remove those lines

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/GluGluHToWW_OffShell/ggWW_off_shell_Cont_loop_sm/ggWW_off_shell_Cont_loop_sm_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+     1        = lpp1    ! beam 1 type 
+     1        = lpp2    ! beam 2 type
+     6500.0     = ebeam1  ! beam 1 total energy in GeV
+     6500.0     = ebeam2  ! beam 2 total energy in GeV
+# To see polarised beam options: type "update beam_pol"
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+     nn23lo1    = pdlabel     ! PDF set                                     
+     230000    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/GluGluHToWW_OffShell/ggWW_off_shell_Cont_loop_sm/ggWW_off_shell_Cont_loop_sm_run_card.dat:47]: same comment here

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/GluGluHToWW_OffShell/ggWW_off_shell_Cont_loop_sm/ggWW_off_shell_Cont_loop_sm_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+#
+#*********************************************************************
+# Store info for systematics studies                                 *
+# WARNING: Do not use for interference type of computation           *
+#*********************************************************************
+   True  = use_syst      ! Enable systematics studies
+#
+systematics = systematics_program ! none, systematics [python], SysCalc [depreceted, C++]
+['--mur=0.5,1,2', '--muf=0.5,1,2', '--pdf=errorset'] = systematics_arguments ! see: https://cp3.irmp.ucl.ac.be/projects/madgraph/wiki/Systematics#Systematicspythonmodule
+# Syscalc is deprecated but to see the associate options type'update syscalc'
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/GluGluHToWW_OffShell/ggWW_off_shell_Cont_loop_sm/ggWW_off_shell_Cont_loop_sm_run_card.dat:281]: same here

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/GluGluHToWW_OffShell/ggWW_off_shell_SBI_loop_sm/ggWW_off_shell_SBI_loop_sm_run_card.dat [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+     1        = lpp1    ! beam 1 type 
+     1        = lpp2    ! beam 2 type
+     6500.0     = ebeam1  ! beam 1 total energy in GeV
+     6500.0     = ebeam2  ! beam 2 total energy in GeV
+# To see polarised beam options: type "update beam_pol"
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+     nn23lo1    = pdlabel     ! PDF set                                     
+     230000    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/GluGluHToWW_OffShell/ggWW_off_shell_SBI_loop_sm/ggWW_off_shell_SBI_loop_sm_run_card.dat:47]: same comment here

**Review by menglu21 — COMMENTED** [review]
besides the comments inline, could you please move the cards to production/2017/13TeV

**Comment by menglu21** [comment]
Hi @sv3048 please remove the initial cards in 13TeV/. as for the fragment, you can add them in https://github.com/cms-sw/genproductions/tree/master/genfragments/ThirteenTeV

**Comment by bbilin** [comment]
Is this a forgotten PR that is replaced/superseeded by another PR?

**Pull request closed without merging**